src/main/java/org/jabref/gui/util/BindingsHelper.java

```diff
@@ -187,6 +188,16 @@ protected String computeValue() {
         return new UiThreadList<>(list);
     }
 
+    public static <T> ObservableValue<T> ifThenElse(ObservableValue<Boolean> condition, T value, T other) {
```

Comment:
Either github does somehow have a problem with this, or it is somehow commented out`, as it's not syntax highlighted